README.md

```diff
@@ -67,13 +67,13 @@ NeosModLoader is licensed under the GNU Lesser General Public License (LGPL). Se
 
 Third-party libraries distributed alongside NeosModLoader:
 
-- [LibHarmony] ([MIT License](https://github.com/pardeike/Harmony/blob/79d62b42c71d005b3cb1d94e741bfd4ce7e03a6b/LICENSE))
+- [LibHarmony] ([MIT License](https://github.com/pardeike/Harmony/blob/master/LICENSE))
 
 Third-party libraries used in source:
 
 - [.NET](https://github.com/dotnet) (Various licenses)
 - [Neos VR](https://neos.com/) ([EULA](https://store.steampowered.com//eula/740250_eula_0))
-- [Json.NET](https://github.com/JamesNK/Newtonsoft.Json) ([MIT License](https://github.com/JamesNK/Newtonsoft.Json/blob/52190a3a3de6ef9a556583cbcb2381073e7197bc/LICENSE.md))
+- [Json.NET](https://github.com/JamesNK/Newtonsoft.Json) ([MIT License](https://github.com/JamesNK/Newtonsoft.Json/blob/master/LICENSE.md))
```

Comment:
Same comment regarding specific license versions.

Reply:
With the Newsoft.Json, the specific version is not pinned anywhere, and the version that Neos uses will be used. So I'm not sure if it makes sense to pin the license to a specific commit?